frontend/src/summary.pug

```diff
@@ -154,23 +158,34 @@
             v-bind:class="{ warn: user.name === '-' }"
           ) {{ user.displayName }} ({{ user.name }})
           .summary-chart__title--contribution.mini-font [{{ user.totalCommits }} lines]
+          a(
+            v-if="filterGroupSelection !== 'groupByRepos'"
```

Comment:
I think you're missing a comma at the end of the line, which is causing the errors from pug-lint
Here is more info on the error: [link](https://github.com/pugjs/pug-lint/blob/master/docs/rules.md#validateattributeseparator-string--object)

Reply:
Oops I forgot to check the travis result. Thanks for the catch :)